bgp/tests/local_tests/BUILD

```diff
@@ -2,7 +2,7 @@ load("@io_bazel_rules_go//go:def.bzl", "go_test")
 
 go_test(
     name = "local_tests_test",
-    size = "medium",
+    size = "large",
```

Comment:
I think local_tests still counts as a medium since it runs for a few minutes right now. I also think this should be run as part of GitHub Actions rather than on cloudbuild. Perhaps add another workflow job that runs the medium jobs?

Reply:
the local test has been flaky on the github action runner, i want to see if solves the issue https://github.com/openconfig/lemming/actions/runs/5860193044/job/15887753577
https://github.com/openconfig/lemming/actions/runs/5686228980/job/15412660877

Reply:
Ok LGTM